src/plugins/intel_gna/tests/deprecated/helpers/tests_file_utils.cpp

```diff
@@ -27,10 +27,10 @@ using namespace std;
 void FileUtils::readAllFile(const std::string &file_name, void *buffer, size_t maxSize) {
     std::ifstream inputFile;
     inputFile.open(file_name, std::ios::binary | std::ios::in);
-    if (!inputFile.is_open()) IE_THROW() << "cannot open file " << file_name;
+    if (!inputFile.is_open()) OPENVINO_THROW("cannot open file ", file_name);
```

Comment:
I am not sure that you really need these changes in this PR, first of all let's concentrate on test utils API.